modules/kmm-using-intree-modules.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="kmm-using-intree-modules_{context}"]
+
+= Using in-tree modules
+
+In some cases, you may want to use only the in-tree module already provided as part of the kernel. The `.spec.moduleLoader` field that defines out-of-tree modules is now optional in the `Module` and `ManagedClusterModule resources`.  
```

Comment:
This part is not accurate. Please see the [m/s docs](https://openshift-kmm.netlify.app/documentation/deploy_kmod/) in the `Supporting Modules without OOT kmods` section.